DX11Starter/ClientSide/ClientManager.cpp

```diff
@@ -257,44 +257,47 @@ void ClientManager::PrepDrawGroup()
             float excess = .5f;
             if (trans.GetEndTime() + excess > time && trans.GetStartTime() <= time)
             {
-                //ItemFromTransHandle(m_drawInfo.m_opaqueObjects[m_drawInfo.m_visibleCount++], trans.GetTransform(time), phenomenas[j].GetHandle());
+               // ItemFromTransHandle(m_drawInfo.m_opaqueObjects[m_drawInfo.m_visibleCount++], trans.GetTransform(time), phenomenas[j].GetHandle());
                 // Projectiles
                 EmitterDrawInfo& drawInfo = m_drawInfo.m_emitters[m_drawInfo.m_emitterCount++];
                 drawInfo.m_handle = phenomenas[j].GetHandle().m_mesh * 2;
                 //AssetManager::get().GetEmitterHandle("Emitter1");
                 drawInfo.startTime = trans.GetStartTime();
                 drawInfo.endTime = trans.GetEndTime() + excess;
+				
 
                 Vector2 trans2;
                 if (trans.GetEndTime() < time)
                 {
                     trans2 = trans.GetPos(trans.GetEndTime());
-                }
+				}
                 else
                 {
                     trans2 = trans.GetPos(time);
-                }
+					ItemFromTransHandle2(m_drawInfo.m_opaqueObjects[m_drawInfo.m_visibleCount++], trans.GetTransform(time), phenomenas[j].GetHandle());
```

Comment:
The spacing looks kind of weird here, I think there might he tabs instead of spaces. I'll take another look later when I'm at my computer.

Reply:
oh yeah it might be when I was typing the other day. Thanks Mark! Just let me know if I could do it from my end. It won't take long.

Reply:
Thats fine, sorry I haven't gotten to it yet. I haven't gotten a good chance.

Reply:
Hey that's completely fine and we understand. I'm free though, just fyi.